packages/web/components/patterns/HighlightBar.tsx

```diff
@@ -149,7 +165,22 @@ function BarContent(props: HighlightBarProps): JSX.Element {
         onClick={() => props.handleButtonClick('comment')}
         css={{ color: '$readerFont', height: '100%', m: 0, p: 0, pt: '6px' }}
       >
-        <CommentIcon size={28} strokeColor={theme.colors.readerFont.toString()} />
+        <HStack css={{ height: '100%', alignItems: 'center' }}>
+          <Note size={24} color={theme.colors.readerFont.toString()} />
```

Comment:
same here for color if you use the css prop (if that's available on Note)

`css={ color: $readerFont }`

Reply:
Oh nevermind Note is a Phosphor component. So you had it right already. Just wanted to point out the shortcut when we're working with a component that supports the css prop 😄

Reply:
love shortcuts 👍  will keep in mind for the future use.